manager/assets/modext/widgets/media/modx.browser.js

```diff
@@ -198,7 +198,7 @@ Ext.extend(MODx.browser.View,MODx.DataView,{
             ,listeners: {
                 'success':{fn:function(r) {
                     if (!Ext.isEmpty(r.object.url)) {
```

Comment:
This checks if there is an URL in the returned result. The value `node.attributes.pathRelative` is always defined and therefore not empty. 

I most likely have to reconsider my approval for this PR. It's better to fix the `modBrowserFileDownloadProcessor` processor and return the right URL so we can use the actual result instead of the "workaround".

Reply:
My gut says this also should be sorted server-side - otherwise, what's the `url` for?